<launch>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="mjpeg" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>

    <!--cam_yaml-->
    <param name="camera_info_url" value="file://$(find autorace_perception)/config/cam_calibration.yaml"/>
  </node>

  <node name="image_proc" pkg="image_proc" type="image_proc" output="screen">
    <remap from="image_raw" to="/usb_cam/image_raw"/>
    <param name="camera_info" value="/usb_cam/camera_info" />
  </node>

  <!-- 시각화 (보정된 영상 확인용) -->
  <node name="image_view" pkg="image_view" type="image_view" output="screen">
    <remap from="image" to="/usb_cam/image_rect_color"/>
    <param name="autosize" value="true" />
  </node>
</launch>

<!--roslaunch autorace_perception usb_cam.launch-->
<!--rqt_image_view /usb_cam/image_rect_color-->